packages/amplication-client/src/util/analytics-events.types.ts

```diff
@@ -166,4 +166,7 @@ export enum AnalyticsEventNames {
 
   // break the monolith preview user
   PreviewUser_GenerateCode = "PreviewUser_GenerateCode",
+
+  ChangedToDotNet = "ChangedToDotNet",
+  ChoseDotNetUsage = "ChoseDotNetUsage",
```

Comment:
is that uses?

Reply:
Good point.
I added the missing analytics.